htdocs/core/class/html.form.class.php

```diff
@@ -7821,6 +7698,8 @@ public function selectForFormsList($objecttmp, $htmlname, $preselectedvalue, $sh
 			if ($tmpfieldstoshow) {
 				$fieldstoshow = $tmpfieldstoshow;
 			}
+		} else if ($objecttmp === 'categorie') {
```

Comment:
I made ~~a copy-paste mistake~~ two copy-paste mistakes.
```suggestion
		} else if ($objecttmp->element === 'category') {
```